src/test/kotlin/uk/gov/justice/digital/hmpps/courtregister/resource/IntegrationTest.kt

```diff
@@ -38,4 +44,23 @@ abstract class IntegrationTest {
     roles: List<String> = listOf(),
     scopes: List<String> = listOf()
   ): (HttpHeaders) -> Unit = jwtAuthHelper.setAuthorisation(user, roles, scopes)
+
+  companion object {
+    private val pgContainer = PostgresContainer.instance
+
+    @JvmStatic
+    @DynamicPropertySource
+    fun properties(registry: DynamicPropertyRegistry) {
+      pgContainer?.run {
```

Comment:
If we had to start a testcontainers instance of Postgres - because we couldn't find it running locally - set the Spring properties to match the testcontainers instance.